deploy/Chart/crds/radius/radapp.io_recipes.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.15.0
+    controller-gen.kubebuilder.io/version: v0.16.0
```

Comment:
It is because of my local controller-gen version. I can revert this change. But before this PR (https://github.com/radius-project/radius/pull/7928) gets merged in we have to do the update. Please see: https://github.com/radius-project/radius/pull/7928#discussion_r1767907825.